src/demo/SingleFileBuild/index.html

```diff
@@ -165,7 +178,7 @@
     }
     const queryWrapperMenuContainer = document.querySelector('#panel1')
     const queryWrapperProps = {
-      token: '',
+      token,
```

Comment:
I think this only worked if people pasted in their session tokens into this code before!